frontend/javascripts/oxalis/api/api_latest.js

```diff
@@ -754,6 +778,16 @@ class TracingApi {
     return getPosition(Store.getState().flycam);
   }
 
+  /**
+   * Sets the current camera position.
```

Comment:
You could also mention `centerPositionAnimated` here which would be the animated version of `setCameraPosition`.